src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1279,6 +1283,14 @@ private void ThrowIfOperationInProgress(Task operationTask, [CallerMemberName] s
                 throw new InvalidOperationException(SR.Format(SR.net_Websockets_AlreadyOneOutstandingOperation, methodName));
             }
         }
+        
+        private void ThrowIfDisposed()
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(GetType().FullName);
```

Comment:
`GetType().FullName` will throw a `MissingMetadataException` if the type has reflection metadata optimized away. `GetType().ToString()` will always work: it will either be the type name, or a string that has the address of a type if the type doesn't have reflection metadata. For cases like this, `ToString()` is definitely the preferred pattern (if nobody else wants to machine-parse the type name later).